noxfiles/ci_nox.py

```diff
@@ -128,6 +130,7 @@ def fidesctl_db_scan(session: nox.Session) -> None:
     """Scan the fidesctl application database to check for dataset discrepancies."""
     session.notify("teardown")
     session.run(*START_APP, external=True)
+    session.run("/bin/sleep", "10", external=True)
```

Comment:
this is going to break on windows. use the native python implementation! something like:

```
import time

time.sleep(10)
```

Reply:
thank you 🙏🏽 I think the output from `nox` suggested using that compared with whatever I had originally. Will swap it out now